src/components/navigation-tabs.ts

```diff
@@ -155,6 +158,16 @@ export class Tabs {
         }).render
       ],
     });
+
+    if (duration !== undefined && duration !== -1) {
```

Comment:
in which case can this be `-1`

Reply:
When its -1, we shouldn't set timeout per documentation.

Reply:
ah I see https://github.com/aws/mynah-ui/blob/main/docs/DATAMODEL.md#notificationprops

Though I don't think this overlay is being created as a notification so I am not sure if the `-1` applies here

Reply:
The idea is keep the same behaviour that we have in notification (I know here we're using directly the Overlay): however, in API perspective, if user send -1, it would be infinite. We can include this in documentation, perhaps.